src/index.tsx

```diff
@@ -31,6 +31,7 @@ render(
         appReducer={appReducer}
         deviceSelect={<BoilerplateDeviceSelector />}
         sidePanel={<SidePanel />}
+        feedback={{}}
```

Comment:
```suggestion
        feedback
```

Reply:
Didn't work here at first. :thinking: Fixed it now.